MomsSpaghetti/Pages/Index.cshtml.cs

```diff
@@ -34,17 +34,22 @@ public class IndexModel : PageModel
         public Like Like { get; set; }
 
         [BindProperty(SupportsGet = true)]
-        public String recipeId { get; set; }
+        public string RecipeId { get; set; }
 
         public IndexModel(ILogger<IndexModel> logger, IMemoryCache cache)
         {
             _logger = logger;
             _cache = cache;
         }
-
+        //Gets the Search Details from the User
```

Comment:
Use proper Intellisense, and be adequately descriptive with comments above a method.